src/optics/GrayAtmosphericStates.jl

```diff
@@ -1,4 +1,57 @@
 
+abstract type AbstractGrayOpticalThickness end
+
+"""
+    GrayOpticalThicknessSchneider2004{FT} <: AbstractGrayOpticalThickness
+
+Optical thickness function parameters from Schneider 2004, J. Atmos. Sci. (2004) 61 (12): 1317–1340.
+DOI: https://doi.org/10.1175/1520-0469(2004)061<1317:TTATTS>2.0.CO;2
+
+# Fields
+$(DocStringExtensions.FIELDS)
+"""
+struct GrayOpticalThicknessSchneider2004{FT} <: AbstractGrayOpticalThickness
+    "lapse rate"
```

Comment:
```suggestion
    "scale height ratio"
```
(that is the physical meaning of this parameter. The lapse rate is controlled by `α`, but is not the name of the parameter.)